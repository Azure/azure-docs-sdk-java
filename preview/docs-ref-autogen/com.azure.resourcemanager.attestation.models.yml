### YamlMime:JavaPackage
uid: "com.azure.resourcemanager.attestation.models"
fullName: "com.azure.resourcemanager.attestation.models"
name: "com.azure.resourcemanager.attestation.models"
summary: "Package containing the data models for AttestationManagementClient. Various APIs for managing resources in\n attestation service. This primarily encompasses per-provider management."
classes:
- "com.azure.resourcemanager.attestation.models.AttestationServiceCreationParams"
- "com.azure.resourcemanager.attestation.models.AttestationServiceCreationSpecificParams"
- "com.azure.resourcemanager.attestation.models.AttestationServicePatchParams"
- "com.azure.resourcemanager.attestation.models.AttestationServiceStatus"
- "com.azure.resourcemanager.attestation.models.JsonWebKey"
- "com.azure.resourcemanager.attestation.models.JsonWebKeySet"
- "com.azure.resourcemanager.attestation.models.OperationsDefinition"
- "com.azure.resourcemanager.attestation.models.OperationsDisplayDefinition"
- "com.azure.resourcemanager.attestation.models.PrivateEndpoint"
- "com.azure.resourcemanager.attestation.models.PrivateEndpointConnectionListResult"
- "com.azure.resourcemanager.attestation.models.PrivateEndpointConnectionProvisioningState"
- "com.azure.resourcemanager.attestation.models.PrivateEndpointServiceConnectionStatus"
- "com.azure.resourcemanager.attestation.models.PrivateLinkServiceConnectionState"
interfaces:
- "com.azure.resourcemanager.attestation.models.AttestationProvider"
- "com.azure.resourcemanager.attestation.models.AttestationProvider.Definition"
- "com.azure.resourcemanager.attestation.models.AttestationProvider.DefinitionStages"
- "com.azure.resourcemanager.attestation.models.AttestationProvider.DefinitionStages.Blank"
- "com.azure.resourcemanager.attestation.models.AttestationProvider.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.attestation.models.AttestationProvider.DefinitionStages.WithLocation"
- "com.azure.resourcemanager.attestation.models.AttestationProvider.DefinitionStages.WithProperties"
- "com.azure.resourcemanager.attestation.models.AttestationProvider.DefinitionStages.WithResourceGroup"
- "com.azure.resourcemanager.attestation.models.AttestationProvider.DefinitionStages.WithTags"
- "com.azure.resourcemanager.attestation.models.AttestationProvider.Update"
- "com.azure.resourcemanager.attestation.models.AttestationProvider.UpdateStages"
- "com.azure.resourcemanager.attestation.models.AttestationProvider.UpdateStages.WithTags"
- "com.azure.resourcemanager.attestation.models.AttestationProviderListResult"
- "com.azure.resourcemanager.attestation.models.AttestationProviders"
- "com.azure.resourcemanager.attestation.models.OperationList"
- "com.azure.resourcemanager.attestation.models.Operations"
- "com.azure.resourcemanager.attestation.models.PrivateEndpointConnection"
- "com.azure.resourcemanager.attestation.models.PrivateEndpointConnection.Definition"
- "com.azure.resourcemanager.attestation.models.PrivateEndpointConnection.DefinitionStages"
- "com.azure.resourcemanager.attestation.models.PrivateEndpointConnection.DefinitionStages.Blank"
- "com.azure.resourcemanager.attestation.models.PrivateEndpointConnection.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.attestation.models.PrivateEndpointConnection.DefinitionStages.WithParentResource"
- "com.azure.resourcemanager.attestation.models.PrivateEndpointConnection.DefinitionStages.WithPrivateEndpoint"
- "com.azure.resourcemanager.attestation.models.PrivateEndpointConnection.DefinitionStages.WithPrivateLinkServiceConnectionState"
- "com.azure.resourcemanager.attestation.models.PrivateEndpointConnections"
metadata: {}
package: "com.azure.resourcemanager.attestation.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-attestation:1.0.0-beta.1
