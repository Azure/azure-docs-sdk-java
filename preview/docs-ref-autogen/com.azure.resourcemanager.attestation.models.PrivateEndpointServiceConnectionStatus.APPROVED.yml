### YamlMime:JavaMember
uid: "com.azure.resourcemanager.attestation.models.PrivateEndpointServiceConnectionStatus.APPROVED*"
fullName: "com.azure.resourcemanager.attestation.models.PrivateEndpointServiceConnectionStatus.APPROVED"
name: "APPROVED"
nameWithType: "PrivateEndpointServiceConnectionStatus.APPROVED"
members:
- uid: "com.azure.resourcemanager.attestation.models.PrivateEndpointServiceConnectionStatus.APPROVED"
  fullName: "com.azure.resourcemanager.attestation.models.PrivateEndpointServiceConnectionStatus.APPROVED"
  name: "APPROVED"
  nameWithType: "PrivateEndpointServiceConnectionStatus.APPROVED"
  summary: "Static value Approved for PrivateEndpointServiceConnectionStatus."
  syntax: "public static final PrivateEndpointServiceConnectionStatus APPROVED"
type: "field"
metadata: {}
package: "com.azure.resourcemanager.attestation.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-attestation:1.0.0-beta.1
